src/site/_drafts/what-should-you-measure-to-improve-performance/index.md

```diff
@@ -0,0 +1,175 @@
+covered in greater detail [here](https://web.dev/fast).   
+Lab data for a first load can be easily acquired through
+[Lighthouse](https://web.dev/fast/discover-performance-opportunities-with-lighthouse),
+while field data (at least for Chrome) is readily available through [Chrome UX
+reports](https://web.dev/fast/chrome-ux-report). A convenient combination of
+both can be found in [PageSpeed
+Insights](https://developers.google.com/speed/pagespeed/insights/). You should
+also track relevant metrics from the field yourself:
+[Measuring these metrics on real users' devices](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#measuring_these_metrics_on_real_users_devices)
+provides a good overview.  
+From a user perspective the most important metrics are:
+
++   **[First Contentful Paint](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#tracking_fpfcp)
+    (FCP)**: The time the user stares at a blank screen. This is
+    when most users bounce, as they don't see progress.
++   **[First Meaningful Paint](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics#tracking_fmp_using_hero_elements)
+    (FMP):** When the user begins to see the main content they came for. This
+    is often the hero image, but for a landing page it may even be a call to
+    action such as a ‘Buy' button, since the user may have arrived with a clear
+    intent (for example, through a targeted ad campaign).
++   **[First Input Delay](https://developers.google.com/web/updates/2018/05/first-input-delay)
+    (FID):** The time the website needs to react to the user's first input.
+    Excessive JavaScript and other asset loading problems can block this,
+    leading to failed taps or clicks, erroneous inputs and page abandonment.
+
+There are more metrics you can look at, but these are a good baseline. Additionally also make sure to capture bounce rates, conversions and user engagement so that you can set these in relation.
+
+## Engagement And Conversion
+
+After the first load of a landing page, a user will proceed through your site,
+hopefully towards a successful conversion.   
+In this phase it is important to have fast and responsive navigations and
+interactions. Unfortunately it is not trivial to measure the complete flow of
+navigation and interaction events in the field, as every user takes different
+paths through the page. We therefore recommend measuring the time needed towards
+conversion or conversion subgoals (‘_Time-to-Action_') by scripting and
+measuring the flow in a lab test, to compare performance over time or with
+competitors. There are two convenient ways of doing this:
+
+### WebPageTest
+
+[WebPageTest](https://www.webpagetest.org/) offers a very flexible scripting
+solution described
+[here](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/scripting).
```

Comment:
Instead of 'here', might consider linkifying the text "a very flexible scripting solution."